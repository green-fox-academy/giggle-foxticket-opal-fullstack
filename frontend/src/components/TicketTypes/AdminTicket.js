import React from 'react';
import './AdminTicket.style.sass';
import GiveIcon from '../DynamicIcon/DynamicIcon';
import Button from '../Button/Button';

function AdminTicket({ title, description , iconName }) {
  console.log(iconName)
  return (
    <div className="container">
      <div className="icon">
{/*        <GiveIcon icon='faAd' />
 */}      </div>
      <div className="content">
        <h1>{title}</h1>
        <p>{description}</p>
      </div>
      <div className="btns">
        <Button buttonStyle="btn--danger--solid">Edit</Button>
        <Button buttonStyle="btn--danger--solid--btn">Delete</Button>
      </div>
    </div>
  );
}
export default AdminTicket;
