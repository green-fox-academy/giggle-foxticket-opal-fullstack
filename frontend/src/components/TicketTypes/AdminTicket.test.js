import React from 'react';
import AdminTicket from './AdminTicket';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <AdminTicket title="Hello" description="bla bla" iconName="FaBeer" />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
