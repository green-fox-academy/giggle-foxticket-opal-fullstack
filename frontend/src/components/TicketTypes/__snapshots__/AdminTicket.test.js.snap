// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="icon"
  />
  <div
    className="content"
  >
    <h1 />
    <p />
  </div>
  <div
    className="btns"
  >
    <button
      className="btn btn--danger--solid btn--medium"
    >
      Edit
    </button>
    <button
      className="btn btn--danger--solid--btn btn--medium"
    >
      Delete
    </button>
  </div>
</div>
`;
